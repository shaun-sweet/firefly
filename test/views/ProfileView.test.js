// import { shallow, createLocalVue } from 'vue-test-utils'
// import Vuex from 'vuex'
// import ProfileView from 'src/views/ProfileView.vue';
// import getters from 'src/store/getters'
// import {initialState, testState} from '../fixtures/state'

// const localVue = createLocalVue()

// localVue.use(Vuex)

// describe('ProfileView', () => {
//   let wrapper;
//   let store;
//   let state;
//   let spy;
//   beforeEach(() => {
//     spy = jest.fn()
//     state = {...testState}
//     store = new Vuex.Store({
//       state,
//       getters
//     })
//     wrapper = shallow(ProfileView,{ store, localVue });
//   })
//   it('should render the view correctly', () => {
//     expect(wrapper.element).toMatchSnapshot();
//   });

// })
