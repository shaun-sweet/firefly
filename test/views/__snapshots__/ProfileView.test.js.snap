// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileView should render the view correctly 1`] = `
<div
  class="layout-padding"
>
  <q-field>
    <q-input
      icon="ion-email"
      stack-label="Full Name"
      type="text"
    />
  </q-field>
   
  <q-field>
    <q-input
      icon="ion-email"
      stack-label="Firefly Address (temp)"
      type="email"
    />
  </q-field>
   
  <q-btn
    color="primary"
    icon="ion-checkmark"
  >
    Save
  </q-btn>
</div>
`;
